contrib/Shopify/sarama/sarama.go

```diff
@@ -114,7 +114,7 @@ type syncProducer struct {
 func (p *syncProducer) SendMessage(msg *sarama.ProducerMessage) (partition int32, offset int64, err error) {
 	span := startProducerSpan(p.cfg, p.version, msg)
 	partition, offset, err = p.SyncProducer.SendMessage(msg)
-	finishProducerSpan(span, partition, offset, err)
+	finishProducerSpan(span, msg, err)
```

Comment:
I'm not sure sarama's SyncProducer modify original message with the valid Partition, Offset .